packages/web/docs/src/pages/product-updates/2025-01-08-static-breaking-change-threshold.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Added a Conditional Breaking Change Static Threshold
+description:
+  A minor improvement to our conditional breaking changes behavior that allows changes to be
+  considered breaking based on a minimum number of requests.
+date: 2025-01-08
+authors: [jdolle]
+---
+
+Expect a lot of improvements to the Hive Console (Schema Registry) in 2025.
+
+One of the first updates this year is a small but potentially very useful option for conditional
+breaking changes. Previously, Conditional Breaking Changes checked compatibility against all
```

Comment:
nit: `Conditional Breaking Changes` could be a link to https://the-guild.dev/graphql/hive/docs/management/targets#conditional-breaking-changes